lib/saml/user.rb

```diff
@@ -85,6 +88,8 @@ def user_attributes_class
         SAML::UserAttributes::DSLogon
       when 'multifactor', 'dslogon_loa3', 'myhealthevet_loa3', LOA::IDME_LOA3, LOA::IDME_LOA1
         SAML::UserAttributes::IdMe
+      when 'ssoe'
+        SAML::UserAttributes::SSOe
```

Comment:
does this take care of the `TODO` above this?

Reply:
👍